src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,35 @@
+package core.basesyntax;
+
+public class Ball {
+    String ballColor;
+    int ballNumber;
```

Comment:
The class name is sufficent to tell us about the object nature. We can use number and color naming here.
```suggestion
    private int number;
```